src/com/hillel/OleksiiMalchev/HW1.java

```diff
@@ -0,0 +1,7 @@
+package com.hillel.OleksiiMalchev;
+
+public class HW1 {
+    public static void main(String[] args) {
+        System.out.println("My Home Work 1");
```

Comment:
Please create separate variable for "My Home Work 1"

Reply:
Done